src/cljs/ataru/hakija/application_handlers.cljs

```diff
@@ -699,6 +699,7 @@
                                                      field-descriptor
                                                      valid?]))}}
         {:db (set-multiple-choice-followup-visibility db field-descriptor option)
```

Comment:
Pitäiskö set-multiple-choice-followup-visibility suorittaa myös rivillä 692, single choicen tapauksessa se suoritetaan sekä question group tapauksessa että muuten?

Reply:
Kysymysryhmässä oleville jutuille ei voi asettaa lisäkysymyksiä. Eli tuo `set-single-choice-followup-visibility` pitäis olla turha tuolla rivillä 733.

Reply:
Ok!